axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -621,7 +621,8 @@
       },
       "confirmations": 1,
       "finality": "finalized",
-      "approxFinalityWaitTime": 1
+      "approxFinalityWaitTime": 1,
+      "decimals": 18
```

Comment:
decimals should be added right after tokenSymbol for clarity. In a separate task, we should probably create a `"token": { .. }` object with relevant details to encapsulate it better